src/test/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidatorTest.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.pay.adminusers.resources;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.Test;
+import uk.gov.pay.adminusers.validations.RequestValidations;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class ServiceUpdateOperationValidatorTest {
+
+    private ObjectMapper mapper = new ObjectMapper();
+    private ServiceUpdateOperationValidator serviceUpdateOperationValidator = new ServiceUpdateOperationValidator(new RequestValidations());
+
+    @Test
+    public void shouldSuccess_whenUpdate_withAllFieldsPresentAndValid() {
```

Comment:
I'm sure this convention and you've copied this as-is but why is it `shouldSuccess` rather than `shouldSucceed` (we use `shouldFail` instead of `shouldFailure`)?

Reply:
Consistency with the shocking grammar used elsewhere!